### DISCUSSION TITLE
using  keepassxc credential helper, works fine cli.  How to get gittyup to stop asking for them?

### DISCUSSION THREAD

**Discussion by dkebler** [Murmele/Gittyup#488] [Q&A]
I have set up this credential hellper.

https://github.com/Frederick888/git-credential-keepassxc

and it works fine pushing repo to my private git server using credentials from my keepass database.

I turned OFF the "store credentials" in gittyup  (as it should? be taken care of from git itself) but it keeps asking for credentials when I push to a private repo.

**are other stores not supported?  or is this a bug?  or am I doing something wrong?**

using  gittytup 1.2.2   on manjaro/arch

git version 2.39.2

the main .gitconfig shows

```
[global]
[credential]
	helper = keepassxc
```

**Comment by dkebler** [comment]
I'm trying to get help via this bug PR which seems to be about my issue

https://github.com/Murmele/Gittyup/pull/464#issuecomment-1458722938

**Comment by exactly-one-kas** [comment]
The problem might be that within a flatpak, the command `git-credential-keepassxc` might not be executable due to the sandbox restrictions and must be started with `flatpak-spawn` on our end

@Murmele can you corroborate this?

**Comment by Murmele** [comment]
@exactly-one-kas yes I will check it out

**Comment by RyanHumps** [comment]
I'm in the same boat as @dkebler. I got keepassxc credential working with Gittyup 1.3.0 on windows but haven't gotten anywhere on Fedora 38. Haven't been able to find a workaround, other than strictly using cli for remotes.

**Comment by RyanHumps** [comment]
For anyone that has the same setup/issue, it appears that the broken link between Gittyup and KeepassXC Credential Helper is due to Native Messaging not working in Snap/Flatpak images. You can find more information [here](https://github.com/keepassxreboot/keepassxc-browser/wiki/Troubleshooting-guide). While there may be workarounds to get the credential helper working when using Gittyup as a Flatpak, I opted to [build it](https://github.com/Murmele/Gittyup#how-to-build) myself and now it works fine.
